spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jms/JmsProperties.java

```diff
@@ -154,6 +154,11 @@ public void setSessionCacheSize(int sessionCacheSize) {
 		 */
 		private Integer maxConcurrency;
 
+		/**
+		 * Timeout to use for receive calls.
```

Comment:
`0` and `-1` have special meaning so it would be nice to have those documented here. See the javadoc of `org.springframework.jms.listener.AbstractPollingMessageListenerContainer#setReceiveTimeout`